devito/function.py

```diff
@@ -1479,7 +1406,7 @@ def _interpolation_indices(self, variables, offset=0):
         for i, idx in enumerate(index_matrix):
             ind_subs = dict([(dim, ind) for dim, ind in zip(self.grid.dimensions, idx)])
             v_subs = [(v, v.subs(ind_subs))
-                      for v in variables if not v.base.function.is_SparseFunction]
+                      for v in variables if not isinstance(v, SparseFunction)]
```

Comment:
why was this necessary?

Reply:
Because `interp/inject` was using `indexify` to make it "easier", now done on functions.